base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -120,7 +120,22 @@ def inheritance_handler_xpath(self, source, specs, inherit_id):
         target_node = self.locate_node(
             source, etree.Element(specs.tag, expr=specs.get('target'))
         )
-        target_node.append(node)
+
+        target_position = specs.get('target_position') or 'inside'
+
+        if target_position == 'after':
+            target_node.addnext(node)
+        elif target_position == 'before':
+            target_node.addprevious(node)
+        elif target_position == 'inside':
```

Comment:
Please support 'move' as a (maybe deprecated) synonym for 'inside'. This module is already used a lot, with the move value. So better not break existing views.

Reply:
This is an additional attribute next to `position='move'` which remains necessary.

Reply:
Yes I see, I was put on a wrong footing by the README, which does not mention target_position.